code/obj/item/clothing/armor.dm

```diff
@@ -434,3 +434,30 @@
 		setProperty("rangedprot", 0.7)
 		setProperty("coldprot", 5)
 		setProperty("heatprot", 35)
+
+/obj/item/clothing/suit/armor/hopuniform
+	name = "burecratic jacket"
+	desc = "A jacket with a bands and stashes you can take on and off as you please"
```

Comment:
spelling: "sashes"
grammar: "a bands and sashes" should either be "a band and sash" or "bands and sashes."

Reply:
got it thanks